examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywall/PaywallScreenState.kt

```diff
@@ -0,0 +1,9 @@
+package com.revenuecat.paywallstester.ui.screens.paywall
+
+import com.revenuecat.purchases.Offering
+
+sealed class PaywallScreenState {
```

Comment:
Love it. I wonder if you'll need to move it to the UI framework to handle these 3 states there instead.